myFitnessTrainer/src/components/EndWorkoutButton.js

```diff
@@ -15,16 +19,26 @@ const EndWorkoutButton = ({ daysCompleted }) => {
             auth.currentUser.email
         );
 
-        const workoutPlanId = await getDocumentId(
+        const workoutPlan = await getDocument(
             "workout_plans",
             "user_id",
             userId
         );
 
+        const workoutPlanId = workoutPlan["id"];
+
         updateDocument("workout_plans", workoutPlanId, {
             days_completed: daysCompleted + 1,
```

Comment:
I created a new user (ID: iknmgLRXZFyONlrOVhDj) and created a workout of 1 day duration and 1 exercise per day. After submitting the exercise, I did not see the "congrats, workout plan completed" message in the console.

```
 LOG  Signed up with: bcd@g.com
 LOG  Document written with ID:  iknmgLRXZFyONlrOVhDj
 LOG  current index 1
 LOG  undefined
 WARN  Possible Unhandled Promise Rejection (id: 0):
TypeError: Cannot read property 'forEach' of undefined
TypeError: Cannot read property 'forEach' of undefined
    at ?anon_0_ (http://192.168.86.249:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:146187:33)
    at next (native)
    at asyncGeneratorStep (http://192.168.86.249:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:4644:26)
    at _next (http://192.168.86.249:19000/nod

https://github.com/osu-cs467-myFitnessTrainer/myFitnessTrainer/assets/21234456/ed8a21ce-c898-4927-9d33-22b419135152

e_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:4663:29)
    at tryCallOne (/private/var/folders/yw/6bx918xn4671rggfcdxz7fph0000gn/T/hermes/build_iphoneos/lib/InternalBytecode/InternalBytecode.js:53:16)
    at anonymous (/private/var/folders/yw/6bx918xn4671rggfcdxz7fph0000gn/T/hermes/build_iphoneos/lib/InternalBytecode/InternalBytecode.js:139:27)
    at apply (native)
    at anonymous (http://192.168.86.249:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:29744:26)
    at _callTimer (http://192.168.86.249:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:29663:17)
    at _callReactNativeMicrotasksPass (http://192.168.86.249:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:29693:17)
    at callReactNativeMicrotasks (http://192.168.86.249:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:29856:44)
    at __callReactNativeMicrotasks (http://192.168.86.249:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:2793:46)
    at anonymous (http://192.168.86.249:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:2605:45)
    at __guard (http://192.168.86.249:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:2777:15)
    at flushedQueue (http://192.168.86.249:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:2604:21)
    at callFunctionReturnFlushedQueue (http://192.168.86.249:19000/node_modules/expo/AppEntry.bundle?platform=ios&dev=true&hot=false:2589:33)
```

https://github.com/osu-cs467-myFitnessTrainer/myFitnessTrainer/assets/21234456/def63cda-92e1-452e-bc5a-9423c4a9c3ab